src/applications/ask-va/containers/DashboardCardsMock.jsx

```diff
@@ -202,6 +207,13 @@ const DashboardCardsMock = () => {
           </va-alert>
         </div>
       )}
+      <VaPagination
+        page={1}
+        pages={3}
+        maxPageListLength={5}
+        showLastPage
+        className="vads-u-border-top--0 vads-u-padding-y--5"
+      />
```

Comment:
Does the pagination work when a page is clicked? I only ask because usually the page change event is handled using `onPageSelect`.

https://design.va.gov/storybook/?path=/docs/uswds-va-pagination--docs

Reply:
We are only mocking this out for an upcoming research study to make things easier for our UX/design team, the real DashboardCards will have the pagination in place with onPageSelect.